packages/remix-eslint-config/rules/typescript.js

```diff
@@ -1,54 +1,49 @@
-const OFF = 0;
-const WARN = 1;
-const ERROR = 2;
-
 module.exports = {
-  "no-dupe-class-members": OFF,
-  "no-undef": OFF,
-
-  // Add TypeScript specific rules (and turn off ESLint equivalents)
-  "@typescript-eslint/consistent-type-assertions": WARN,
-  "@typescript-eslint/consistent-type-imports": WARN,
-
-  "no-array-constructor": OFF,
-  "@typescript-eslint/no-array-constructor": WARN,
+  // TODO: These rules might be nice to enable... we should investigate eventually!
+  "@typescript-eslint/ban-ts-comment": "off",
+  "@typescript-eslint/ban-types": "off",
+  "@typescript-eslint/no-empty-function": "off",
+  "@typescript-eslint/no-empty-interface": "off",
+  "@typescript-eslint/no-explicit-any": "off",
+  "@typescript-eslint/no-inferrable-types": "off",
+  "@typescript-eslint/no-namespace": "off",
+  "@typescript-eslint/no-non-null-assertion": "off",
+  "@typescript-eslint/no-var-requires": "off",
+  "no-var": "off",
+  "prefer-rest-params": "off",
 
-  // There is a bug w/ @typescript-eslint/no-duplicate-imports triggered
-  // by multiple imports inside of module declarations. We should reenable
-  // this rule when the bug is fixed.
-  // https://github.com/typescript-eslint/typescript-eslint/issues/3071
-  "no-duplicate-imports": OFF,
-  // "@typescript-eslint/no-duplicate-imports": WARN,
```

Comment:
Is this something we could enable now that the referenced bug is closed?  It doesn't look to be enabled in the recommended rules

Reply:
The rule is deprecated in favor of [import/no-duplicates](https://github.com/import-js/eslint-plugin-import/blob/HEAD/docs/rules/no-duplicates.md). I'd recommend just using that one instead.